lib/KeyboardAwareHOC.js

```diff
@@ -180,6 +207,48 @@ function listenToKeyboardEvents(ScrollableComponent: React$Component) {
       }, keyboardOpeningTime)
     }
 
+    scrollIntoView = async (
+      element: React.Element<*>,
+      options: ScrollIntoViewOptions = {}
+    ) => {
+      if (!this._rnkasv_keyboardView || !view) {
+        return
+      }
+
+      const [
+        parentLayout,
+        childLayout
+      ] = await Promise.all([
+        this._measureElement(this._rnkasv_keyboardView),
+        this._measureElement(element)
+      ])
+
+      const getScrollPosition = options.getScrollPosition || this._defaultGetScrollPosition
+      const { x, y, animated } = getScrollPosition(parentLayout, childLayout, this.position)
+      this._container.scrollToPosition(x, y, animated)
```

Comment:
Does `scrollToPosition` return a promise / cancellable resource? If so we should return the result of the call. Sorry, I'm not intimate with the API. 

```js
return this._container.scrollToPosition(x, y, animated)
```